pkg/options/options.go

```diff
@@ -97,6 +99,7 @@ func (o *Options) AddFlags() {
 	o.flags.Var(&o.NamespacesDenylist, "namespaces-denylist", "Comma-separated list of namespaces not to be enabled. If namespaces and namespaces-denylist are both set, only namespaces that are excluded in namespaces-denylist will be used.")
 	o.flags.Var(&o.MetricAllowlist, "metric-allowlist", "Comma-separated list of metrics to be exposed. This list comprises of exact metric names and/or regex patterns. The allowlist and denylist are mutually exclusive.")
 	o.flags.Var(&o.MetricDenylist, "metric-denylist", "Comma-separated list of metrics not to be enabled. This list comprises of exact metric names and/or regex patterns. The allowlist and denylist are mutually exclusive.")
+	o.flags.Var(&o.OptInMetrics, "opt-in-metrics", "Comma-separated list of metrics which are opt-in and not enabled by default. This is in addition to the metric allow- and denylists")
```

Comment:
Should it be called metric-opt-in-list or similar then?

Reply:
Maybe we could also just call it --metric-enable-experimental (to enable metrics that are experimental) and have users filter it on the denylist then?

Reply:
"experimental" does not mean / imply the same as "opt-in" does, as metrics which are opt-in are metrics that can be stable but might not be applicable for everyone or only to a small subset of use-cases and thus shouldn't be enabled by default.

I thought about the "--metric-opt-in-list" as well" but I feel like "--opt-in-metrics" reads and pronounces a lot more natural than "--metric-opt-in-list" does, as "allowlist" is more of a verb and "opt-in" more of a noun which indicates the action of "opting-in for a metric", though I understand what you mean (to keep some kind of similarity in the naming of CLI options).

Reply:
@fpetkovski, what do you think about this? I wouldn't say to rename the option to `--metric-enable-experimental` (because of the aforementioned reason), but I am in between about @mrueg's suggestion for `--metric-opt-in-list` (or something similar) versus `--opt-in-metrics`.

Reply:
Using the same naming convention would also indicate that the flags are related and control the same functionality.

Reply:
Will change it to `--opt-in-metrics` then. I'm here to contribute value to the project, not to nitpick on a name because I like it, hahah. In favor of consistency as well, as long as it does not affect the clarity of its purpose.

Thanks @mrueg and @fpetkovski for the thoroughness! 😉